File: examples/java/kora-java-resilient/src/main/java/io/koraframework/example/resilient/MyCircuitBreaker.java
package io.koraframework.example.resilient;

import io.koraframework.resilient.circuitbreaker.CircuitBreaker;
import io.koraframework.resilient.circuitbreaker.annotation.CircuitBreakerSpec;

@CircuitBreakerSpec("resilient.circuitbreaker.my_cb")
public interface MyCircuitBreaker extends CircuitBreaker {}

File: examples/kotlin/kora-kotlin-resilient/src/main/kotlin/io/koraframework/kotlin/example/resilient/MyCircuitBreaker.kt
package io.koraframework.kotlin.example.resilient

import io.koraframework.resilient.circuitbreaker.CircuitBreaker
import io.koraframework.resilient.circuitbreaker.annotation.CircuitBreakerSpec

@CircuitBreakerSpec("resilient.circuitbreaker.my_cb")
interface MyCircuitBreaker : CircuitBreaker
